README.rst

```diff
@@ -16,7 +16,7 @@ Prerequisites
 Create setup with below system requirements and install required packages on
 host from which this repo is used as library or tests are executed.
 
-System Requirements
+Minimum System Requirements
 ===================
```

Comment:
Allign this line with text

Reply:
This is just a seperation for heading and body not actual decorator for heading.  This line wont be visible in readme.